transformer_engine/pytorch/graph.py

```diff
@@ -158,11 +200,12 @@ def _make_graphed_callables(
     # from ending up in any captures.
     torch.cuda.synchronize()
     with torch.cuda.stream(torch.cuda.Stream()):
-        for c_i, func in enumerate(callables):
-            args = sample_args[c_i]
-            static_input_surface = per_callable_static_input_surfaces[c_i]
+        for func_idx, func in enumerate(callables):
+            args = sample_args[func_idx]
+            kwargs = sample_kwargs[func_idx]
+            static_input_surface = per_callable_static_input_surfaces[func_idx]
             for _ in range(num_warmup_iters):
-                outputs, _ = _tree_flatten(func(*args))
+                outputs, _ = _tree_flatten(func(*args, **kwargs))
```

Comment:
All elements in outputs should be a tensor since in below it is checked with `requires_grad`. However not all outputs of the function may be tensors. Would it be possible to filter the outputs to only tensors?
Pytorch uses [pytree.tree_leaves ](https://github.com/NVIDIA/TransformerEngine/pull/945/files#diff-867908fd35970c62b0147b524cf76ce7173d41ac235101ca59c24a6d88979c6aR208) though I'm not sure if that would help.

Reply:
Please disregard I misunderstood sorry, it seems only tensor inputs/outputs are supported at the moment.